test/integration/testdata/gen-plugin-e2e-configmap.golden

```diff
@@ -34,7 +34,7 @@ spec:
     value: "true"
   - name: E2E_EXTRA_ARGS
     value: --progress-report-url=http://localhost:8099/progress
-  image: k8s.gcr.io/conformance:v123.456.789
+  image: k8s.gcr.io/conformance:$SONOBUOY_K8S_VERSION
```

Comment:
The k8s_version wasnt applied; this is because `gen plugin e2e` was using its own logic.

PR is pending for that.

Reply:
https://github.com/vmware-tanzu/sonobuoy/pull/1350